app/templates/edit.hbs

```diff
@@ -57,17 +57,17 @@
   <:aside>
     {{#if this.editor}}
       <Sidebar as |Sidebar|>
-        <Sidebar.Collapsible @title='Insert'>
-          <ArticleStructurePlugin::ArticleStructureCard 
+        <Sidebar.Collapsible @title='Insert' @expandedInitially={{true}}>
```

Comment:
Nitpicky, but maybe we could change this to `isOpenInitially` to be consistent with the Appuniversum Card

Reply:
we could, but in this case, "eh"

go for it if you feel like it